src/plugins/dashboard/public/dashboard_listing/dashboard_listing.tsx

```diff
@@ -223,6 +224,7 @@ export const DashboardListing = ({
             notifications,
             overlays,
             http,
+            theme,
```

Comment:
We've got this on our roadmap, but likely we won't be able to prioritize it in the near future because it will unfortunately be a very large change. I agree though that this is important.

Reply:
Ok thanks Devon 👍